CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Guidelines for Contributing Code
+
+New Relic welcomes code contributions by the Node community to this module, and
+have taken effort to make this process easy for both contributors and our
+development team.
+
+## Process
+
+### Feature Requests
+
+Feature requests should be submitted in the [Issue tracker](../../issues), with
+a description of the expected behavior & use case. Before submitting an Issue,
+please search for similar ones in the [closed
+issues](../../issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement).
+
+### Pull Requests
+
+We can only accept PRs for version v4.0.0 or greater due to open source
+licensing restrictions.
+
+### Code of Conduct
+
+Before contributing please read the [code of conduct](./CODE_OF_CONDUCT.md)
+
+Note that our [code of conduct](./CODE_OF_CONDUCT.md) applies to all platforms
+and venues related to this project; please follow it in all your interactions
+with the project and its participants.
+
+### Contributor License Agreement
+
+Keep in mind that when you submit your Pull Request, you'll need to sign the
+CLA via the click-through using CLA-Assistant. If you'd like to execute our
+corporate CLA, or if you have any questions, please drop us an email at
+opensource@newrelic.com.
+
+For more information about CLAs, please check out Alex Russell’s excellent
+post, [“Why Do I Need to Sign
+This?”](https://infrequently.org/2008/06/why-do-i-need-to-sign-this/).
+
+### Slack
+
+We host a public Slack with a dedicated channel for contributors and
+maintainers of open source projects hosted by New Relic. If you are
+contributing to this project, you're welcome to request access to the
+\#oss-contributors channel in the newrelicusers.slack.com workspace. To request
+access, see https://newrelicusers-signup.herokuapp.com/.
+
+## PR Guidelines
+
+### Version Support
+
+When contributing, keep in mind that New Relic customers (that's you!) are running many different versions of Node, some of them pretty old. Changes that depend on the newest version of Node will probably be rejected, especially if they replace something backwards compatible.
```

Comment:
Looks like the text above is line wrapped at 80-100 characters, something like that, and then here it's longer lines. Just thought I'd point it out.

Reply:
I can update to make it consistent. 👍